From 3cabe02b7847befca00c7705bf2ccdf6761fcb7e Mon Sep 17 00:00:00 2001
From: Alain Volmat <alain.volmat@foss.st.com>
Date: Mon, 24 Mar 2025 15:55:53 +0100
Subject: [PATCH] arm64: dts: st: use st,stm32mp21-dcmipp compatible on
 stm32mp211.dtsi

Switch to usage of the stm32mp21 dedicated compatible for dcmipp in
stm32mp211.dtsi in order to enable usage of the histogram block.

Change-Id: Ia387f83aa59cbcbe47bdd194257c3389478e7d34
Signed-off-by: Alain Volmat <alain.volmat@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/446245
Domain-Review: Philippe CORNU <philippe.cornu@foss.st.com>
Tested-by: Alain VOLMAT <alain.volmat@st.com>
Reviewed-by: Alain VOLMAT <alain.volmat@st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
Reviewed-by: Philippe CORNU <philippe.cornu@foss.st.com>
---
 arch/arm64/boot/dts/st/stm32mp211.dtsi | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/st/stm32mp211.dtsi b/arch/arm64/boot/dts/st/stm32mp211.dtsi
index e1e4f781e2ae..7da7a37f4c2b 100644
--- a/arch/arm64/boot/dts/st/stm32mp211.dtsi
+++ b/arch/arm64/boot/dts/st/stm32mp211.dtsi
@@ -2706,7 +2706,7 @@ soc1: soc@1 {
 		ranges = <0x0 0x0 0x0 0x80000000>;
 
 		dcmipp: dcmipp@48030000 {
-			compatible = "st,stm32mp25-dcmipp";
+			compatible = "st,stm32mp21-dcmipp";
 			reg = <0x48030000 0x1000>;
 			interrupts = <GIC_SPI 170 IRQ_TYPE_LEVEL_HIGH>;
 			resets = <&rcc DCMIPP_R>;
-- 
2.50.1

